docs/dg/projectManagement.md

```diff
@@ -21,6 +21,26 @@ This page contains information about project management tasks. The target audien
 
 <!-- ==================================================================================================== -->
 
+## Making a release on Github
+
+Before making a release, please check the following prerequisite.
+
+* Ensure that you have a proper version of **JDK `1.8.0`** installed (==Not **JDK `11`** or other later version==).
+* Ensure that the `JAVA_HOME` environment variable is correctly set to your JDK installation directory. You can refer to the guide [here](https://docs.oracle.com/cd/E19182-01/821-0917/inst_jdk_javahome_t/index.html).
```

Comment:
We should avoid having links that just says "here" or "click here" ([see this guide](https://www.w3.org/QA/Tips/noClickHere) for more information). The link ideally should represent an action that you want to direct the user to take.

Reply:
Thanks for the suggestion. I will change it accordingly.